microprofile/tests/arquillian/src/main/java/io/helidon/microprofile/arquillian/HelidonMethodExecutor.java

```diff
@@ -43,10 +53,10 @@
 
     /**
      * Invoke method after enrichment.
-     *
+     * <p>
```

Comment:
Please fix configuration of your IDE, it inserts <p> instead of empty new lines in javadocs

Reply:
:smile:  fixed